packages/integrations/cloudflare/README.md

```diff
@@ -48,7 +48,9 @@ Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode
 
 For most projects the adapter default of `advanced` will be sufficient; the `dist` folder will contain your compiled project. Switching to directory mode allows you to use [pages plugins](https://developers.cloudflare.com/pages/platform/functions/plugins/) such as [Sentry](https://developers.cloudflare.com/pages/platform/functions/plugins/sentry/) or write custom code to enable logging.
 
-In directory mode the adapter will compile the client side part of your app the same way, but moves the worker script into a `functions` folder in the project root. The adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
+In directory mode the adapter will compile the client side part of your app depending on the new `build.split` setting. If `split` is disabled the adapter compiles everything in the same way, but moves the worker script into a `functions` folder in the project root. The adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control.
```

Comment:
```suggestion
In directory mode, the adapter will compile the client side part of your app the same way by default, but moves the worker script into a `functions` folder in the project root. In this case, the adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control.
```
P. S. We don't refer to things as new, because that "gets old" quickly!  :smile: 

And, *especially* if this is intended to be experimental, but even if not, I would tend to instead just describe the above behaviour as "default" (since by default, `split` will not be true). The paragraph below can describe what happens if someone has actively configured `split: true`.